src/node_env_var.cc

```diff
@@ -353,6 +354,8 @@ MaybeLocal<Object> CreateEnvVarProxy(Local<Context> context,
   env_proxy_template->SetHandler(NamedPropertyHandlerConfiguration(
       EnvGetter, EnvSetter, EnvQuery, EnvDeleter, EnvEnumerator, data,
       PropertyHandlerFlags::kHasNoSideEffect));
-  return scope.EscapeMaybe(env_proxy_template->NewInstance(context));
+  MaybeLocal<Object> env_proxy = env_proxy_template->NewInstance(context);
+  CHECK(env_proxy.ToLocalChecked()->SetPrototype(context, Null(isolate)).FromJust());
+  return scope.EscapeMaybe(env_proxy);
```

Comment:
It doesn't really make sense to use .ToLocalChecked() (and .FromJust()) and still return a MaybeLocal; I would prefer to do error handling here rather than crashing on failure, and keep returning an empty MaybeLocal if a V8 operation leads to an exception

Reply:
Fixed - thanks for the help :)